### PR TITLE
Update readme file

### PR DESCRIPTION
**Pull request by izmalk**
Update the main Readme file. Fix obvious issues and add relevant information to bring closer to the Charmed Kafka readme.

### CODE REVIEW

**Review by theoctober19th — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,19 +1,50 @@
...
+
+Security issues in the Charmed Spark K8s bundle can be reported through [LaunchPad](https://wiki.ubuntu.com/DebuggingSecurity#How%20to%20File). Please do not file GitHub issues about security issues.
+
 ## Contributing
 
-Canonical welcomes contributions to the `spark8t` toolkit. Please check out our [guidelines](python/CONTRIBUTING.md) if you're interested in contributing to the solution. Also, if you truly enjoy working on open-source projects like this one and you would like to be part of the OSS revolution, please don't forget to check out the [open positions](https://canonical.com/careers/all) we have at [Canonical](https://canonical.com/).  
+Canonical welcomes contributions to the Charmed Spark bundle. Please check out our [contribution guidelines](python/CONTRIBUTING.md) if you're interested in contributing to the solution. If you truly enjoy working on open-source projects like this one and you would like to be part of the OSS revolution, please don't forget to check out the [career opportunities](https://canonical.com/careers/all) we have at [Canonical](https://canonical.com/).
```
Comment (deusebio) [inline README.md:44]: **niptick** Maybe worth adding a reference to the CLA?

**Review by deusebio — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (izmalk) [reply README.md:44]: I don't think it is needed since we have the link in the `python/CONTRIBUTING.md` file.

**Review by welpaolo — APPROVED** [review]
LGTM

**Pull request merged**